src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -303,12 +303,20 @@ PyTypeObject JSObjectItemsProxyType = {
 };
 
 static void cleanup() {
+  // Clean up the PythonMonkey module
+  Py_XDECREF(PythonMonkey_Null);
+  Py_XDECREF(PythonMonkey_BigInt);
+
+  // Clean up SpiderMonkey
   delete autoRealm;
   delete global;
   if (GLOBAL_CX) JS_DestroyContext(GLOBAL_CX);
```

Comment:
should we set GLOBAL_CX = null here, since it is not longer usable/useful after the context is destroyed?

Reply:
Done in https://github.com/Distributive-Network/PythonMonkey/pull/443/commits/5b9deec8ec071d5c86749b99cb413fa6ea5d8ab1